cps/kni.c

```diff
@@ -911,32 +911,52 @@ rd_send_batch(struct cps_config *cps_conf, struct mnl_nlmsg_batch *batch,
 {
```

Comment:
(I can't comment on the prototype of `rd_send_batch()`.) Add `const` to parameter `cps_conf` of `rd_send_batch()`.